setup.cfg

```diff
@@ -59,3 +56,63 @@ markers =
     xfail_backends
     xfail_unsupported
     xpass_backends
+
+[mypy]
```

Comment:
are all of these actually failing? really?

Reply:
there is a list of the errors (not updated) here: https://github.com/ibis-project/ibis/pull/2161#issuecomment-611590282